README.md

```diff
@@ -43,7 +43,7 @@ source-control-host = "gitlab"
 The default content generated is the following:
 
 ```
-Found a bug? <a href="/link/to/edit/page">Edit this file on GitHub.</a>
+Found a bug? <a href="/link/to/edit/page">Edit this file on Gitlab.</a>
```

Comment:
`GitHub` is still the default even if the repo is hosted on Gitlab, cf. https://github.com/wonderfulspam/mdbook-open-git-repo/blob/master/src/lib.rs#L13

Whether this should be the case is up for discussion but it means this crate is a drop-in replacement for `open-on-gh`. Simply change those five characters in your `book.toml` and this plugin works the same as the parent (provided your repo is hosted on Github, that is. If it is hosted on Gitlab, the parent crate would silently fail).

I _was_ considering making the `GitHub|GitLab` part variable based on the source control host but I couldn't be arsed. PR welcome!

I suggest we change this back to the actual default but change the link to something slightly more realistic.
```suggestion
Found a bug? <a href="https://git.host.com/repo/project/branch/edit/chapter.md">Edit this file on GitHub.</a>
```